Content.Shared/Abilities/Milira/WingFlightComponent.cs

```diff
@@ -0,0 +1,162 @@
+    public bool InertiaActive;
+
+    /// <summary>
+    /// Максимальный множитель масштаба спрайта во время полёта.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float MaxScaleMultiplier = 1.35f;
+
+    /// <summary>
+    /// Минимальный множитель масштаба, когда визуал полёта отсутствует.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float MinScaleMultiplier = 1.0f;
+
+    /// <summary>
+    /// Текущий множитель масштаба, синхронизируемый с клиентами.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float CurrentScaleMultiplier = 1.0f;
+
+    /// <summary>
+    /// Скорость приближения масштаба к целевому значению.
+    /// </summary>
+    [DataField]
+    public float ScaleLerpRate = 4f;
+
+    /// <summary>
+    /// Минимальная доля выносливости (0..1) для активации.
+    /// </summary>
+    [DataField]
+    public float ActivationThreshold = 0.5f;
+
+    /// <summary>
+    /// Разовый расход выносливости при запуске полёта.
+    /// </summary>
+    [DataField]
+    public float ActivationStaminaDamage = 20f;
+
+    /// <summary>
+    /// Постоянный расход выносливости в секунду при активном полёте.
+    /// </summary>
+    [DataField]
+    public float SustainStaminaPerSecond = 5f;
+
+    /// <summary>
+    /// Доля выносливости (0..1), при которой полёт выключается принудительно.
+    /// </summary>
+    [DataField]
+    public float AutoDisableThreshold = 0.25f;
+
+    /// <summary>
+    /// Момент завершения инерции (только сервер).
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly), AutoPausedField]
+    public TimeSpan? InertiaEndTime;
+
+    /// <summary>
+    /// Аккумулятор для периодического расхода выносливости (только сервер).
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public float SustainAccumulator;
+
+    /// <summary>
+    /// Применялся ли суффикс полёта, чтобы при выключении вернуть маркинг обратно.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public bool AppliedMarkingOnEnable;
+
+    /// <summary>
+    /// Список оригинальных маркингов для восстановления после отключения полёта (только сервер).
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public Dictionary<int, string> OriginalMarkings = new();
+
+    /// <summary>
+    /// Исходные коллизии фикстур, чтобы при завершении полёта вернуть проход блокирующих тайлов.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public Dictionary<string, int> OriginalCollisionMasks = new();
+
+    /// <summary>
+    /// Исходные слои фикстур, сохраняются вместе с масками для симметричного возврата.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public Dictionary<string, int> OriginalCollisionLayers = new();
+
+    /// <summary>
+    /// Оригинальный масштаб спрайта (только клиент, для визуализации).
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public Vector2? OriginalScale;
```

Comment:
```suggestion
    [ViewVariables]
    public bool AppliedMarkingOnEnable;

    /// <summary>
    /// Список оригинальных маркингов для восстановления после отключения полёта (только сервер).
    /// </summary>
    [ViewVariables]
    public Dictionary<int, string> OriginalMarkings = new();

    /// <summary>
    /// Исходные коллизии фикстур, чтобы при завершении полёта вернуть проход блокирующих тайлов.
    /// </summary>
    [ViewVariables]
    public Dictionary<string, int> OriginalCollisionMasks = new();

    /// <summary>
    /// Исходные слои фикстур, сохраняются вместе с масками для симметричного возврата.
    /// </summary>
    [ViewVariables]
    public Dictionary<string, int> OriginalCollisionLayers = new();

    /// <summary>
    /// Оригинальный масштаб спрайта (только клиент, для визуализации).
    /// </summary>
    [ViewVariables]
    public Vector2? OriginalScale;
```

VV и так имеются ReadOnly по умолчанию

Reply:
сделал